### DISCUSSION TITLE
Alignment doesn't seem right on Stat / Gauge / Wide

### DISCUSSION THREAD

**Discussion by itsxaos** [h2oai/wave#1443] [Q&A] [answered]
The alignment of the gauge on the wide gauge card doesn't seem right, it moves when the value / auxiliary value changes, and generally doesn't look like it does in the [docs](https://wave.h2o.ai/docs/examples/stat-wide-gauge).

Here's what it looks like, on the left in Vivaldi (Chromium) and Firefox on the right:

[video](https://user-images.githubusercontent.com/33079230/170277424-57ecad0d-1f11-41f3-b0a8-8c1ccf3e9483.mp4)

I'm not sure how to go about fixing this, would appreciate your help!

**Comment by mturoci** [comment, marked as answer]
Hi @itsxaos, thanks for your interest in Wave!

> it moves when the value / auxiliary value changes

This is something we should fix on our side.

> generally doesn't look like it does in the [docs](https://wave.h2o.ai/docs/examples/stat-wide-gauge).

Can you elaborate? They seem identical to me.

**Reply by itsxaos** [reply]
Hi!

Yes, sorry, I should have been more precise. Generally it does look like the docs, but specifically the alignment of the gauge and the text isn't the same.
I took a closer look at it, and it's not just the alignment, the circle is actually a little bigger, too.

I recreated the card that's pictured in the docs and lined them up, you can see that there is a lot more empty space around the circle in the recreation.
![gauge_alignment](https://user-images.githubusercontent.com/33079230/170470567-4ba78ffe-89cb-46ba-a6a9-3bacdb014675.png)

This is the example:
```
#!/usr/bin/env python

from h2o_wave import site, ui

del site['/test']
page = site['/test']
page.add('example', ui.wide_gauge_stat_card(
	box = '1 1 2 1',
	title = 'Dash',
	value = '$21.60',
	aux_value = '55.34%',
	plot_color = '$red',
	progress = 0.5534
))
page.save()
```

Maybe you could check whether the way it looks for me is how it's supposed to look now, and it's just the docs that are outdated?

**Reply by mturoci** [reply]
Oh, I see, the example screenshot is outdated. We suggest looking at [widgets docs section](https://wave.h2o.ai/docs/widgets/stat/wide_gauge_stat/) instead as it has the most up to date screenshot always.

**Reply by itsxaos** [reply]
Ah, ok, that makes sense!
I wasn't aware of that section of the docs, I've just been looking at the examples section.

Personally, I sort of prefer the old look aesthetically, and it seems like it would handle longer value/aux value text better.
Still, I can see the benefit to having the gauge itself be a little larger and for the values in the example (i.e. fairly short text) it's arguably more balanced with the new alignment.

Either way, thanks for clearing that up, and thanks for making Wave!

**Reply by mturoci** [reply]
The main reason is this new implementation is responsive what may come in handy when using [flex layouts](https://wave.h2o.ai/docs/layout/#flex-layout). The old one had its dimensions hardcoded.

The moving problem is tracked in #1446
